README.md

```diff
@@ -1,6 +1,6 @@
 # Compose Cupertino
 
-[![compose-mp-version](https://img.shields.io/badge/compose--multiplatform-1.5.11-blue)](https://github.com/JetBrains/compose-multiplatform)
+[![compose-mp-version](https://img.shields.io/badge/compose--multiplatform-1.6.0--rc01-blue)](https://github.com/JetBrains/compose-multiplatform)
```

Comment:
Is it a Compose last released version is compiled with. Rollback it please

Reply:
Ah, of course. Will revert this change.